pkgs/servers/mail/smtp-to-telegram/default.nix

```
@@ -0,0 +1,21 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  name = "smtp-to-telegram";
+  rev = "8092fc365b902d160d3e3de6ceb7ca67a17392d0";
+  vendorSha256 = "sha256-q7v2IQqWHap4JSmRcXjSI6aWb3kF/tA1dk5NXWBeytE=";
+
+  src = fetchFromGitHub {
+    inherit rev;
```

```suggestion
  pname = "smtp-to-telegram";
  version = "unstable-XXXX-XX-XX";

  vendorSha256 = "sha256-q7v2IQqWHap4JSmRcXjSI6aWb3kF/tA1dk5NXWBeytE=";

  src = fetchFromGitHub {
    rev = "8092fc365b902d160d3e3de6ceb7ca67a17392d0";
```
XXXX-XX-XX should be the date of the commit.

Also vendorSha could be moved after src to be in a logical order.

---

done 🙁